src/main/java/org/assertj/core/api/AbstractStandardSoftAssertions.java

```diff
@@ -36,6 +36,7 @@
    * @param actual the actual value.
    * @return the created assertion object.
    */
+  @CheckReturnValue
```

Comment:
Note that you can also apply `@CheckReturnValue` to classes (or packages) which has the same affect as annotation all of the elements. If you go that route, Error Prone defines a [`@CanIgnoreReturnValue`](http://errorprone.info/api/latest/com/google/errorprone/annotations/CanIgnoreReturnValue.html) annotation you can use to opt-out specific methods.